README.md

```diff
@@ -63,6 +63,28 @@ This integer specifies the LUKS key size (in bytes).
 ##### `encryption_luks_version`
 This integer specifies the LUKS version to use.
 
+##### `vdo_deduplication`
+This specifies whether or not the Virtual Data Optimizer will be used.
+When set, duplicate data stored on underlying storage volume will be
+deduplicated resulting in more storage capacity.
+Can be used together with `vdo_compression`(optional) and `vdo_logical_size` (strongly recommended).
+Limit one `storage_volume` per `storage_pool`.
+Underlying volume has to be at least 9 GB.
```

Comment:
Minimum size for VDO pool should be 5 GiB, but that's bare minimum so 9 GiB is probably ok here.

Reply:
Fixed: Added a note about ~5GiB bare minimum limit.